rs/protobuf/def/registry/crypto/v1/crypto.proto

```diff
@@ -80,9 +80,20 @@ message SchnorrKeyId {
   string name = 2;
 }
 
+enum VetKdCurve {
+  VETKD_CURVE_UNSPECIFIED = 0;
+  VETKD_CURVE_BLS12_381_G2 = 1;
+}
+
+message VetKdKeyId {
+  VetKdCurve curve = 1;
+  string name = 2;
+}
+
 message MasterPublicKeyId {
   oneof key_id {
     EcdsaKeyId ecdsa = 1;
     SchnorrKeyId schnorr = 2;
+    VetKdKeyId vetkd = 3;
```

Comment:
is this PR forward and backward compatible? 
is it possible replica to contain such variants in the state when it is rolled out?
to prevent that should it first the proto changes to be roll out and only then the management canister types?

Reply:
after offline discussion we agreed it does not get into replicated state and also there's no code that uses this field yet.

Reply:
I added a respective section on _Registry Protobuf backwards-compatibility_ in the PR description.